lib/pgBackRest/Backup/File.pm

```diff
@@ -223,21 +246,21 @@ sub backupFile
     #
     # If the file was checksummed then get the size in all cases since we don't already have it.
     if ((($iCopyResult == BACKUP_FILE_COPY || $iCopyResult == BACKUP_FILE_RECOPY) &&
-            $oStorageRepo->driver()->capability(STORAGE_CAPABILITY_SIZE_DIFF)) ||
+            $oStorageRepo->capability(STORAGE_CAPABILITY_SIZE_DIFF)) ||
         $iCopyResult == BACKUP_FILE_CHECKSUM)
     {
-        $lRepoSize = ($oStorageRepo->info(STORAGE_REPO_BACKUP . "/${strBackupLabel}/${strFileOp}"))->size();
+        $lRepoSize = ($oStorageRepo->info(STORAGE_REPO_BACKUP . "/${strBackupLabel}/${strFileOp}"))->{size};
     }
 
     # Return from function and log return values if any
     return logDebugReturn
     (
         $strOperation,
-        {name => 'iCopyResult', value => $iCopyResult, trace => true},
-        {name => 'lCopySize', value => $lCopySize, trace => true},
-        {name => 'lRepoSize', value => $lRepoSize, trace => true},
-        {name => 'strCopyChecksum', value => $strCopyChecksum, trace => true},
-        {name => 'rExtra', value => $rExtra, trace => true},
```

Comment:
In the Perl code, this only worked if *all* the params were set to trace, so this was really a noop.  Anyway, I thought this function deserved to be at debug level (which is actually was because of the incomplete trace options).

Reply:
Hmmm, this one did have all the traces.  It must have been another place I was thinking of.  In any case debug level is better here.